### ISSUE TITLE
Fly command

### ISSUE THREAD

**Issue by SmilerRyan** [Overv/WebCraft#40]
Currently to reach places high up, you must build, jump and delete blocks after to get to a location.
How would I add a /fly command, which simply disables falling when users press (or hold) a key (ex. f)?

**Comment by cymno** [comment]
Like this: a5e1d17e024baddb68b90e2843529312c28cca40
Btw, the movement code in ./js/player.js is very nice to play around with and modify. You could easily e.g. add a sprint option by finding out how to detect a "shift" press in javascript and then increase the speed [here](https://github.com/Overv/WebCraft/blob/47b3f6910864ea0d35ed203e1e337aa958fad606/js/player.js#L235) accordingly.

Here are the steps I took to actually run the project (a bit hidden imo):
```
$git clone this-repo
$npm install socket.io express
$node server
```
Then visit http://localhost:3000/multiplayer.html or http://localhost:3000/singleplayer.html in a browser (or replace _localhost_ by your ip-adress)

**Comment by SmilerRyan** [comment]
Thanks, however this has only partially solved my solution as I cannot move any other direction except for up/down while holding the key to fly.

**Comment by cymno** [comment]
Yes, for that it is necessary to enable [in-air movement](https://github.com/Overv/WebCraft/blob/a5e1d17e024baddb68b90e2843529312c28cca40/js/player.js#L223). This also makes the jump more responsive, as it is possible to adjust the direction after the jump has already started. If you'd really like to keep the old system it's also possible to enable in-air movement only when "f" is pressed.
